### PR TITLE
#76 add spark.sql.shuffle.partitions to DefaultSparkConfiguration

### LINKED ISSUE #76: Add spark.sql.shuffle.partitions setting to the DefaultSparkConfiguration
## Background
In an effort to lower the run time of tests, add a setting for `spark.sql.shuffle.partitions`. The default is 200 and makes no sense for spark-based unit tests.

## Feature
`.config("spark.sql.shuffle.partitions", "1")` to the `DefaultSparkConfiguration`

## Additional info
I ran the tests a few times. It is not life-saving but it is something.

### Spark commons
**Without the setting** 
sbt test  492.02s user 39.22s system 742% cpu 1:11.54 total
**With the setting**
sbt test  305.25s user 18.57s system 556% cpu 58.167 total

### Enceladus spark-jobs
**Without the setting** 
mvn test  881.18s user 87.80s system 327% cpu 4:56.18 total
**With the setting**
mvn test  713.31s user 74.34s system 324% cpu 4:02.44 total

### PR DESCRIPTION
**Pull request by Zejnilovic**
Closes #76

### CODE REVIEW

**Review by lsulak — APPROVED** [review]
Of course, it makes sense. 

I just read the code, not pulled/compiled/ran tests since this is a very small change code-wise.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1 @@
-* @Zejnilovic
-* @dk1844
-* @benedeki
-* @lsulak
+* @Zejnilovic @dk1844 @benedeki @lsulak
```
Comment (benedeki) [inline .github/CODEOWNERS:1]: Why this change? Seems to me one person per line is easier to navigate. 🤔

Comment (Zejnilovic) [reply .github/CODEOWNERS:1]: Because it is syntactically wrong, it is a rule per line. Meaning that this is read as 

- Owner of everything is Zejnilovic
- Owner of everything is dk1844
- Owner of everything is benedeki
- Owner of everything is lsulak

And the rules are overwriting each other, with the latest rule having priority. So @lsulak being the only code owner.

This is nice for custom rules but not for `*` rules. 

Example of a smidge more complex `CODEOWNERS`
```
# Everything is owned by all of these guys
* @Zejnilovic @dk1844 @benedeki @lsulak

# Except for this, the test part is owned by miroslavpojer
spark-commons-test* @miroslavpojer
```

Comment (Zejnilovic) [reply .github/CODEOWNERS:1]: image

See the current state of this PR. Only lsulak is owner having the little shield next to his name.

Comment (benedeki) [reply .github/CODEOWNERS:1]: Thanks for the explanation 🙇

**Review by benedeki — APPROVED** [review]

**Comment by dk1844** [comment]
The implementation LGTM (correctly follows the issue's description), but I am entirely sure about [the issue's decision (# partitions = 1)](#76).

**Review by dk1844 — APPROVED** [review]
LGTM (just read the code)
